oeps/oep-0014-proc-archive-repos.rst

```diff
@@ -77,19 +77,11 @@ Archive Steps
 
 These steps should be followed for all repos within the edX organization(forks included). After some experiments with keeping archived repos in the ``edx`` organization, we've learned that having abandoned code show up in searches hinders work to understand the current state of the system and the risk around new work, particularly deprecations and API changes. Thus we decided to move all archived repositories to a separate org.
 
-1. Update the README.rst file in the repository to state that it is archived, using the `README Archive Statement`_ below.
+1. (Recommended) Update the README.rst file in the repository to state that it is archived, using the `README Archive Statement`_ below.
 
-2. Update the openedx.yaml file, creating it if necessary:
+2. Unless you have the relevant permissions to the work in this step, create an IT help ticket and ask them to do the following:
 
-   - Add ``archived: True``.
-
-   - Remove the ``openedx-release`` key if it is present.
-
-   - It is not necessary for the openedx.yaml file to define an owner for archived repos.
-
-3. Unless you have the relevant permissions to the work in this step, create an IT help ticket and ask them to do the following:
-
-   - Update the description of the repository to begin with ``[ARCHIVED]``
+   - Update the description of the repository to begin with ``[ARCHIVED]`` (TODO: Do we do this or need to???)
```

Comment:
**Author Concern:** Do we do this and do we need to?

Reply:
I think we can remove this -- marking the repo as archived in the settings should be sufficient.

Reply:
Note that at the time this was originally written, Github had not yet [released](https://github.blog/2017-11-08-archiving-repositories/) their archive setting.